lib/docker.ts

```diff
@@ -54,16 +54,18 @@ class Docker {
    * Runs the command, catching any expected errors and returning them as normal
    * stderr/stdout result.
    */
-  public async runSafe(cmd: string, args: string[] = []) {
+  public async runSafe(
+    cmd: string,
+    args: string[] = [],
+    // no error is thrown if any of listed errors is found in stderr
+    ignoreErrors: string[] = ["No such file", "file not found"],
```

Comment:
Consider declaring and using a constant for the default array value

Reply:
Here I'd err to the side of simpler explicit code at the cost of slight duplication (errors handling logic is a bit surprising), but in any way I hope this code will be ditched soon for static analysis! :)